### DISCUSSION TITLE
issue: I can't build

### DISCUSSION THREAD

**Discussion by franmc01** [react-hook-form/react-hook-form#9667] [Q&A]
### Version Number

7.41

### Codesandbox/Expo snack

none

### Steps to reproduce

1. Install dependency
2. yarn build

### Expected behaviour

![image](https://user-images.githubusercontent.com/50973629/209363242-3db75f35-74b9-4c30-8686-abcef2c15f2d.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
https://github.com/react-hook-form/react-hook-form/issues/2848

**Comment by Moshyfawn** [comment]
I'm pretty sure there's nothing RHF can do and it's your webpack config that needs to be updated anyway

**Comment by bluebill1049** [comment]
If you can reproduce this issue, we can see if we can help, but most likely as Moshyfawn suggested, this is probably a build config issue.
